hail/python/hailtop/batch/docs/api.rst

```diff
@@ -102,6 +102,23 @@ at `<https://batch.hail.is>`__.
     backend.ServiceBackend
 
 
+aiotools
+~~~~~~~~
```

Comment:
I agreed, but I like the idea of formatted documentation of our developer APIs. Is it possible to create a fourth entry in "Reference (Python API)" that's like "Developers Only" and then under that we just have all of hailtop *except* fs and batch?

We can also defer this to a different PR and just keep aiotools hidden until then.

Reply:
i only included this because having `FileValidator` be a base class for the backends that were in the docs already made the build fail unless they had autodocs included as well, but since the refactor got rid of the base class approach, i removed this as well